go.mod

```diff
@@ -247,3 +247,6 @@ replace github.com/openshift/api => github.com/openshift/api v0.0.0-202208231438
 replace github.com/terraform-providers/terraform-provider-nutanix => github.com/nutanix/terraform-provider-nutanix v1.5.0
 
 replace github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
+
+// https://bugzilla.redhat.com/show_bug.cgi?id=2064702
+replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
```

Comment:
This `replace` is needed for any indirect dependencies we have, but the direct dependency we have in the first `require` block is a different, earlier, version than this... just curious did you try bumping that direct dependency?

Long term we should be able to remove this `replace` once all the indirect dependencies no longer use the bad version of the crypto package.

Reply:
I'm hoping that by only adding the `replace` this PR will be easily cherry-pickable to 4.11, the version against which the CVEs were reported.
I'll be happy to revisit this for 4.12 in a separate PR.

Reply:
@patrickdillon I rebased on top of current master and commit 1dd74ca7a8bcd2978d056bf742e9cff409f5fbd0 took care of bumping the x/crypto package.